README.md

```diff
@@ -16,20 +14,67 @@ npm install dostroy
 ```
 Example usage to limit number of requests from a remote address to a Node.js server:
 ```
-const dostroy = require('dostroy')
-const express = require('express')
-const app = express()
-
 config = {
   rateLimiting: true,
-  logging: false
+  dynamicRateLimiting: true,
+  userActiveTimeout: 10000,
+  requestLimit: 10,
+  requestInterval: 10000,
+  rudy: true
 }
 
-app.use(dostroy(config))
+const server = app.listen(3000)
+const dostroyConfig = dostroy.init(server, config)
+app.use(dostroy.protect(dostroyConfig))
+```
+## Config
+Dostroy allows for some degree of configuration. The individual mitigations can be turned on or off and have their thresholds set. If no config is passed, Dostroy will employ all mitigations with standard values.
```

Comment:
Maybe add a section with default values since we mention them? Or write them in their sections below